widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidgetContent.kt

```diff
@@ -115,6 +116,13 @@ fun BalanceSection(
     }
 }
 
+// Utility function to trim unnecessary zeros after the decimal point
+private fun trimBalanceZeros(balance: String): String {
```

Comment:
Thank you for your valuable feedback. I've carefully considered your recommendation and made the necessary adjustments directly in the AmountFormatting.kt utility class. Rather than modifying WalletBalanceWidgetContent.kt.
In the updated code, I've ensured that the formatting for amounts below 1000 is handled correctly. which was not present there.